extensions/data-transfer/portability-data-transfer-apple/src/main/java/org/datatransferproject/datatransfer/apple/photos/AppleMediaInterface.java

```diff
@@ -369,7 +369,8 @@ public static NewMediaRequest createNewMediaRequest(
       @Nullable final String mediaType,
       @Nullable final String encodingFormat,
       @Nullable final Long creationDateInMillis,
-      @Nullable final String singleFileUploadResponse) {
+      @Nullable final String singleFileUploadResponse,
+      @Nullable final Boolean isFavorite) {
```

Comment:
nit: could we switch to a java primtiive everywhere you're storing a local `isFavorite` state?

```suggestion
      final boolean isFavorite) {
```

Reply:
This was done.